### PR TITLE
Add translation for Indonesian languange

### PR DESCRIPTION
**Pull request by muhibbudins**
Hi @iamkun,

I try to add translation for Indonesian Language with key name **id**, and then i add some condition at .gitignore file like this :

```
# build
locale/
!src/locale/
plugin/
dayjs.min.js
```

To ignore only **locale/** folder not **src/locale/**.

Thank you

### CODE REVIEW

**Comment by iamkun** [comment]
Thanks @muhibbudins .

However, I think, `locale/` should ignore only `root/locale/` not other deep `locale` folder?

**Comment by muhibbudins** [comment]
Urwell @iamkun ,

Yups, but i don't know why on my VSCode and Terminal they ignore every `locale/` folder haha

File: src/locale/id.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  S: 'detik',
+  MIN: 'menit',
+  H: 'jam',
+  D: 'hari',
+  W: 'minggu',
+  M: 'bulan',
+  Q: 'kuartal',
+  Y: 'tahun',
+  DATE: 'tanggal',
+  weekdays: 'Minggu_Senin_Selasa_Rabu_Kamis_Jum\'at_Sabtu'.split('_'),
```
Comment (iamkun) [inline src/locale/id.js]: should this be Jumat ? I don't know, just feel weird

**Comment by iamkun** [comment]
Besides, I'm going to merge pr#189, and there's a little change in our locale file.
Would you please update your translation a bit to fit our new code? 

You could check the `diff` in `src/locale/es.js` locale here:
https://github.com/iamkun/dayjs/pull/189/files

File: src/locale/id.js (hunk 1 above)
Comment (muhibbudins) [reply src/locale/id.js]: Hmmm okey i'll fix it

**Comment by muhibbudins** [comment]
Hi @iamkun ,

I'm already update based `src/locale/es.js` on #189.

Thank you

**Comment by iamkun** [comment]
Great THX

**Comment by muhibbudins** [comment]
Urwell @iamkun 
Thanks for awesome project 😃

**Comment by iamkun** [comment]
🎉 This PR is included in version 1.6.4 🎉@muhibbudins

**Pull request merged**